components/ws-daemon/pkg/resources/controller_test.go

```diff
@@ -54,43 +54,43 @@ func TestControlCPU(t *testing.T) {
 			Name:      "bucket regular use",
 			Opts:      []ControllerOpt{WithCPULimiter(bktLimiter())},
 			Consumer:  splitpointConsumer(fixedConsumer(500), randomConsumer(20, 100), 5*time.Minute),
-			Validator: aucValidator(49985),
+			Validator: aucValidator(53385),
```

Comment:
Assuming the period stays the same in our tests (100ms), and the units remain the same (mCPU), why do those numbers change?

Reply:
fixed